app/services/account_reset/grant_requests_and_send_emails.rb

```diff
@@ -48,5 +49,17 @@ def grant_request_and_send_email(arr)
       end
       true
     end
+
+    def fraud_wait_period_not_met?(arr, now)
+      if (arr.user.fraud_review_pending? ||
+        arr.user.fraud_rejection?) &&
+         fraud_wait_period_days.present?
+        return arr.requested_at > (now - fraud_wait_period_days.days)
+      end
+    end
+
+    def fraud_wait_period_days
+      IdentityConfig.store.account_reset_fraud_user_wait_period_days
+    end
```

Comment:
This seems like the sort of thing we might want to incorporate into the existing `AccountReset::GrantRequest` service class, as there's already some validation happening there.

Secondarily, we may want to rearrange the condition to take advantage of boolean short-circuiting on the configuration value, to avoid unnecessary queries for the user.